src/media/css/buttons.styl

```diff
@@ -93,9 +93,10 @@ a.button {
     }
 }
 
-.arrow-button-left,
-.arrow-button-right {
+.arrow-button {
     background-color: $action-positive;
+    background-image: url(../img/pretty/arrow-l.svg);
```

Comment:
Yes I plan to change it there also. We don't need two image requests for prev/next and made the base class a bit more reusable.